<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据抓取工具</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>抓取设置</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="dataTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>911</width>
     <height>531</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="queryData">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemId">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入检测项</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lotId">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入批号</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="mainLog">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>670</y>
     <width>911</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="productId">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入产品型号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dataExport">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>数据导出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
